src/services/serviceCommons.js

```diff
@@ -1,5 +1,5 @@
 const defaultFeatherMongooseOptions = {
-  multi: ['patch'],
+  multi: ['patch', 'remove'],
```

Comment:
We get an error in this line because we have blocked multiple delete (I saw it when I was reading the logs to fix this bug), so added `remove` to the whitelist
https://github.com/Giveth/feathers-giveth/blob/f245076fdab506f8e914e66ed1b491ca6fc82dbd/src/blockchain/lib/eventHandlerQueue.js#L26-L30